events/uk/london_rscds.yaml

```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=../../events_schema.json
+events:
+  - name: London RSCDS Chelsea beginners and improvers
+    links:
+      - "https://www.rscdslondon.org.uk/category/branchactivities/branchclasses/beginnersandimprovers/"
+    start: 2023-01-11T19:00:00+00:00
+    end: 2023-01-11T21:00:00+00:00
+    country: UK
+    city: London
+    styles: [reeling, scd]
```

Comment:
Are you sure this is reeling as well as SCD?

Reply:
Good point, it's a fine line but this one is probably not.